docs/privacy-policy.md

```diff
@@ -0,0 +1,82 @@
+# Privacy Policy
+
+#### _Effective:_ February 8, 2022
+
+
+This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.
+
+1. [Collection Of Information](#Collection-Of-Information)
+
+2. [Use Of Information](#Use-Of-Information)
+
+3. [Sharing Of Information](#Sharing-Of-Information)
+
+4. [Additional Details](#Additional-Details)
+
+5. [References](#References)
+
+6. [Contact Us](#Contact-Us)
+
+
+
+## Collection Of Information
+
+We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend the Operate First Data Science Community Meetups:
+1. Your name
+
+    *a.* as it appears on your Google Account, in case you join through your Google Account; or
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:
```

Comment:
Google analytics collects a lot of information such as location of the user, number of times they logged in, which page they spent the most time at etc., PTAL at a [google analytics privacy policy template.](https://www.termsfeed.com/public/uploads/2022/01/sample-google-analytics-privacy-policy-template.pdf) maybe we can borrow something from here.

Reply:
Thanks Shrey, will look into this :+1: